testament/categories.nim

```diff
@@ -688,8 +689,22 @@ proc runJoinedTest(r: var TResults, cat: Category, testsDir: string) =
 
 # ---------------------------------------------------------------------------
 
-proc processCategory(r: var TResults, cat: Category, options, testsDir: string,
+proc loadSkipFrom(name: string): seq[string] =
```

Comment:
Shouldn't this returns a HashSet[string] instead?

Reply:
no, we iterate below using a substring search, not an exact match

Reply:
Ok and why is that?

Reply:
so that when the test name contains variants (like the gc tests), you can skip all of them together, or a whole category, or all tests with markandsweep on etc

I thought about regex at first but this seems like a good start. if ever this becomes a perf problem, one could build a composite matcher.

Reply:
Meh. Ok.